recipe/meta.yaml

```diff
@@ -21,18 +22,22 @@ requirements:
     - python >=3.7
     - geopandas-base >=0.10
```

Comment:
@jakirkham grayskull changed "geopandas-base" to "geopandas", so I assume that then with the auto-update for a new version, this would happen each time again?

Is there a way to prevent that? Or does that mean enabling grayskull might not work for this repo?

Reply:
Sorry for missing this @jorisvandenbossche 😞

Maybe it is worth [raising a Grayskull issue]( https://github.com/conda/grayskull/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D+Title+of+the+bug ) to note this and discuss a solution?